docs/core/building/distribution-packaging.md

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
avoid ing on file and folder names for SEO purposes (we were not following guidelines earlier). probably packaging is ok in this scenario but building should be build